### DISCUSSION TITLE
记得之前某个版本首页过滤可以隐藏任意版块，现在怎么变了

### DISCUSSION THREAD

**Discussion by pbxs666** [the1812/Bilibili-Evolved#1839] [综合讨论 / General]
脚本一直在用，都是有新版就更新。但最近几次没看过更新日志，不知道这个功能什么时候改变了，是哪个版本？

**Comment by the1812** [comment]
现在不也可以吗
<img width="400" src="https://user-images.githubusercontent.com/26504152/117913083-48ad4e80-b313-11eb-810f-87342b17d328.png">

**Reply by Geeyun-JY3** [reply]
这边试了试，好像确实有些问题：
![image](https://user-images.githubusercontent.com/49396421/118019151-3cfd6e80-b38b-11eb-9123-1b6d0fbd1c98.png)

**Reply by the1812** [reply]
不太对劲(
我之后修一下
